src/models/notificationThreads.ts

```diff
@@ -0,0 +1,43 @@
+import { DataTypes, Model } from 'sequelize';
+import { sequelize } from '@/sequelize-instance';
+import type { CreationOptional, InferAttributes, InferCreationAttributes } from 'sequelize';
+
+export class NotificationThread extends Model<InferAttributes<NotificationThread>, InferCreationAttributes<NotificationThread>> {
+	declare id: CreationOptional<number>;
+	declare user_id: string;
+	declare thread_id: string;
+	declare updated: CreationOptional<Date>;
+	declare created: CreationOptional<Date>;
+}
+
+NotificationThread.init({
+	id: {
+		type: DataTypes.INTEGER,
+		autoIncrement: true,
+		primaryKey: true
+	},
+	user_id: {
+		type: DataTypes.STRING
+	},
```

Comment:
It could be, but since this is a separate model it's worth having it's own id. However, since there should only be one per user I'll add a unique constraint